doc/classes/Image.xml

```diff
@@ -521,6 +528,7 @@
 			<description>
 				Saves the image as a WebP (Web Picture) file to a byte array. By default it will save lossless. If [param lossy] is true, the image will be saved lossy, using the [param quality] setting between 0.0 and 1.0 (inclusive). Lossless WebP offers more efficient compression than PNG.
 				[b]Note:[/b] The WebP format is limited to a size of 16383×16383 pixels, while PNG can save larger images.
+				[b]Note:[/b] This method is only available in engine builds with the WEBP module enabled. By default, the WEBP module is enabled, but it can be disabled at build-time using the [code]module_webp_enabled=no[/code] SCons option.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the WEBP module enabled. By default, the WebP module is enabled, but it can be disabled at build-time using the [code]module_webp_enabled=no[/code] SCons option.
```